src/pages/write-for-egghead/index.mdx

```diff
@@ -0,0 +1,237 @@
+export const meta = {
+  title: `Writing for egghead`,
+}
+
+import FancyGuide from 'layouts/fancy-guide'
+import {TopicListComponent} from './topics.js'
+import RainbowMDX from './mdx-demo.js'
+
+export default ({children}) => (
+  <FancyGuide frontMatter={meta}>{children}</FancyGuide>
+)
+
+<ProseSection>
+
+
+## Trying to get better at technical writing and explaining your ideas clearly?
+
+Writing for egghead is a great way to explore new topics, flex your skills, and get helpful guidance along the way.
+
+Anyone can write for egghead, you just need to let us know you're up for it <span>✋🏾</span>
+
+<a href="/write-for-egghead#pitch-form">
+  <button className="mt-4 flex items-center sm:px-5 px-3 py-2 rounded-md border-1 focus:border-blue-600 focus:outline-none bg-blue-600 border-blue-800 text-blue-100">
+    Join our Writing Crew
+  </button>
+</a>
+
+## Topic Ideas
+
+We're currently prioritising React articles, but we're always open to a wide variety of ideas.
+
+### Browse Our Topic Garden <span>🌱</span>
+
+</ProseSection>
+
+
+<TopicListComponent />
```

Comment:
@MaggieAppleton have you thought about linking to the Notion document instead? This seems to be a _living_ document that can be managed/modified a lot easier on Notion. Similar to how CSS Tricks does it.

Reply:
I personally think it's pretty nice to have them inline, but also see the advantages of this suggestion.

Reply:
Inline would be nicer since it's less confusing for the reader (vs. having to open a new window in a different context) if it wouldn't be too much friction for us to have the repo be the 'living document'. 

But for the moment will swap it back to Notion - haven't figured out how to format it on here in a way that's not awkward. Works for now 👍